app/components/e-textarea.vue

```diff
@@ -0,0 +1,277 @@
+              :name="name"
+              :disabled="disabled"
+              v-bind="$attrs"
+              @focus="onFocus"
+              @blur="onBlur"
+              @input="onInput">
+    </textarea>
+    <span v-if="!hasDefaultState && !hasFocus" :class="b('icon-splitter')"></span>
+    <div v-if="notification && hasFocus" :class="b('notification')">
+      <c-form-notification :state="state" v-html="notification"/>
+    </div>
+  </div>
+</template>
+
+<script>
+  import formStates from '@/mixins/form-states';
+  import CFormNotification from '@/components/c-form-notification';
+
+  export default {
+    name: 'e-textarea',
+    components: {
+      CFormNotification
+    },
+    mixins: [formStates],
+    inheritAttrs: false,
+
+    props: {
+      /**
+       * Value passed by v-model
+       */
+      value: {
+        default: null,
+        type: String,
+      },
+
+      /**
+       * Adds name attribute
+       */
+      name: {
+        required: true,
+        type: String,
+      },
+
+      /**
+       * Adds a notification block in a state container below the textarea
+       */
+      notification: {
+        type: String,
+        default: null
+      }
+    },
+    // data() {
+    //   return {};
+    // },
+
+    computed: {
+
+      /**
+       * Defines state modifier classes
+       *
+       * @returns  {Object}   BEM classes
+       */
+      modifiers() {
+        return {
+          ...this.stateModifiers,
+          notification: Boolean(this.$props.notification && this.hasFocus)
+        };
+      },
+
+      /**
+       * Checks if the component is in default state.
+       *
+       * @returns {Boolean}
+       */
+      hasDefaultState() {
+        return this.state === 'default';
+      }
+    },
+    // watch: {},
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+    // mounted() {},
+    // beforeUpdate() {},
+    // updated() {},
+    // activated() {},
+    // deactivated() {},
+    // beforeDestroy() {},
+    // destroyed() {},
+
+    methods: {
+      /**
+       * Emits input to parent component
+       *
+       * @param   {String}  event   Field input
+       */
+      onInput(event) {
+        this.$emit('input', event.target.value);
+      },
+
+      /**
+       * Emits focus to parent and wrapper component. This is needed for e.g. setting the focus on the label as well.
+       * Updates "hasFocus" state
+       */
+      onFocus() {
+        this.hasFocus = true;
+
+        this.$emit('focus');
+        this.$parent.$emit('focus');
+      },
+
+      /**
+       * Emits blur to parent and wrapper component. This is needed for e.g. removing the focus on the label as well.
+       * Updates "hasFocus" state
+       */
+      onBlur() {
+        this.hasFocus = false;
+
+        this.$emit('blur');
+        this.$parent.$emit('blur');
+      }
+    }
+    // render() {},
+  };
+</script>
+
+<style lang="scss">
+  $e-textarea-height: 77px;
```

Comment:
- [ ] I would not define a fixed height, better use a $prop for rows and add the html attribute "rows" to the textarea element